wolfcrypt/src/port/Renesas/renesas_tsip_aes.c

```diff
@@ -35,16 +35,33 @@
 
 #include <wolfssl/wolfcrypt/wc_port.h>
 #include <wolfssl/wolfcrypt/error-crypt.h>
-
+#include <wolfssl/internal.h>
 #include <wolfssl/wolfcrypt/aes.h>
 #include "wolfssl/wolfcrypt/port/Renesas/renesas-tsip-crypt.h"
 
+
+#define TSIP_AES_GCM_AUTH_TAG_SIZE  16
+
+typedef e_tsip_err_t (*aesGcmEncInitFn)
+        (tsip_gcm_handle_t*, tsip_aes_key_index_t*, uint8_t*, uint32_t);
+typedef e_tsip_err_t (*aesGcmEncUpdateFn)
+        (tsip_gcm_handle_t*,uint8_t*, uint8_t*, uint32_t, uint8_t*, uint32_t);
+typedef e_tsip_err_t (*aesGcmEncFinalFn)
+        (tsip_gcm_handle_t*, uint8_t*, uint32_t*, uint8_t*);
+typedef e_tsip_err_t (*aesGcmDecInitFn)
+        (tsip_gcm_handle_t*, tsip_aes_key_index_t*, uint8_t*, uint32_t);
+typedef e_tsip_err_t (*aesGcmDecUpdateFn)
+        (tsip_gcm_handle_t*,uint8_t*, uint8_t*, uint32_t, uint8_t*, uint32_t);
+typedef e_tsip_err_t (*aesGcmDecFinalFn)
+        (tsip_gcm_handle_t*, uint8_t*, uint32_t*, uint8_t*, uint32_t);
+
+
 struct Aes;
```

Comment:
Do we need this `struct Aes` definition here?  It seems like we should see this from including `<wolfssl/wolfcrypt/aes.h>`?

Reply:
Please address

Reply:
Removed it.